### PR TITLE
feat(docs): added web components template info

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
None.

### Description

Adding information and links to the Web Components Template.

### Changelog

**New**

- added Web Components template information and links

### CODE REVIEW

**Comment by shixiedesign** [comment]
Should this information be added to Web Components tab instead of React tab? 
Currently: src/pages/developing/frameworks/react.mdx
Suggesting: src/pages/developing/frameworks/web-components.mdx

Maybe replacing this part with the new content you added? Just a suggestion. Lemme know if it makes sense. FYI @janchild 
image

File: src/pages/developing/frameworks/web-components.mdx [hunk 1]
```diff
@@ -160,9 +160,12 @@ View the `<dds-dotcom-shell>` in [Storybook Web Components](http://www.ibm.com/s
 </Column>
```
Comment (janchild) [inline src/pages/developing/frameworks/web-components.mdx:160]: You could remove line 74 `Available tags:` 
The paragraph above does a great job of introducing the table. :) 

Lines 82 and 95
Should it be `Here are examples of setting by tag:` 
Suggestion to replace `on` with `of` unless I'm not understanding something. 

Line 101
Suggestion to remove `what is` from the sentence. 
Check npmjs.com to see ~what is~ the latest available version. 

Line 138
I'd change the link to appear as follows, and add a period to the end. 
Learn more about <Link to="/developing/carbon-cdn-style-helpers">Carbon CDN style helpers</Link>.
Okay, that's not showing... I was trying to link only the words `Carbon CDN style helpers` in the sentence.

**Review by janchild — COMMENTED** [review]
Hi @IgnacioBecerra! For some reason, I wasn't able to do inline editing but I added some comments. Let me know if you need me to explain anything. 

This page is great! I really like it. I only had a few little suggestions. :)

**Review by janchild — APPROVED** [review]
Excellent! 🚀

**Pull request merged**